### ISSUE TITLE
[Link and button labels] Button and link language isn't descriptive and/or their purpose isn't clear. (08.03.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#60371]
#### VFS team name
Digitize Forms

#### VFS product name
Form 21-4142(a)

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Button and link language isn't descriptive and/or their purpose isn't clear.

### Issue Details
On [Step 7](https://staging.va.gov/supporting-forms-for-claims/release-information-to-va-form-21-4142/records-requested) when multiple treatment records have been entered, the treatment records are displayed in condensed form with the name of the provider and an edit button for editing that record. Each edit button has the same `aria-label`, "Edit Treatment record."

Each button should have a unique accessible name, which in this case means the `aria-label` should be unique (or as unique as possible) so that screen reader users can identify which treatment record they're editing.

### VA.gov Experience Standard
[Category Number 08, Issue Number 03](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4_A

### Platform Recommendation
Easiest option here is for the `aria-label` to be "Edit [name of provider]" or something similar. It doesn't solve for the situation where a user is entering multiple treatment records from the same provider so it's possible for the buttons to not have unique names. But `Edit [name of provider]" would be a comparable experience to the visual cues provided to sighted users.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by gmrabian** [comment]
[PR for fix](https://github.com/department-of-veterans-affairs/vets-website/pull/24581). Added the index to the aria label so it will say `Edit treatment record 1` `Edit treatment record 2` etc. At least it distinguishes it for now, even though we can't read out actual data from that record to make it more clear.
I also updated the Update and Remove buttons to include an index in their aria labels

**Comment by jeana-adhoc** [comment]
This looks great, thank you @gmrabian. Confirmed on staging! Closing ticket.
